com.unity.robotics.urdf-importer/Tests/Runtime/Extensions/UrdfGeometryTests.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.TestTools;
+using RosSharp.Urdf;
+
+namespace RosSharp.Urdf.Tests
+{
+    public class UrdfGeometryTests
+    {
+        Link.Geometry box;
+        Link.Geometry cylinder;
+        Link.Geometry sphere;
+
+        [SetUp]
+        public void SetUp()
+        {
+            box = new Link.Geometry(box: new Link.Geometry.Box(new double[] { 2, 3, 4 }));
+            cylinder = new Link.Geometry(cylinder: new Link.Geometry.Cylinder(1, 4));
+            sphere = new Link.Geometry(sphere: new Link.Geometry.Sphere(1));
+        }
+
+        [Test]
+        public void ExportGeometryData_Box()
+        {
+            var parent = new GameObject("Parent").transform;
+            UrdfGeometryCollision.Create(parent, GeometryTypes.Box);
+            var t = parent.transform.Find("Box");
+            Assert.IsNotNull(UrdfGeometry.ExportGeometryData(GeometryTypes.Box, t));
+        }
+
+        [Test]
+        public void ExportGeometryData_Cylinder()
+        {
+            // Force runtime mode to avoid creating asset file
+            RuntimeURDF.runtimeModeEnabled = true;
+            var parent = new GameObject("Parent").transform;
+            UrdfGeometryCollision.Create(parent, GeometryTypes.Cylinder);
+
+            var t = parent.transform.Find("Cylinder");
+            Assert.IsNotNull(UrdfGeometry.ExportGeometryData(GeometryTypes.Cylinder, t));
```

Comment:
Might be better if you could check if the exported geometry is equal to a `new Link.Geometry`; similar to the `ExporterGeometryData_Sphere()`